packages/downloads/src/portal/build-existing-exports-portal-query.ts

```diff
@@ -0,0 +1,48 @@
+import { SearchQueryBuilder } from "@esri/arcgis-rest-portal";
+import { parseDatasetId } from "@esri/hub-common";
+
+/**
+ * Builds the Portal API query string to search for exports from a given dataset
+ *
+ * @param datasetId - The dataset ID
+ * @param options - A set of options including requested export types and spatialRefId
+ * @returns
+ */
+export function buildExistingExportsPortalQuery(
+  datasetId: string,
+  options: {
+    formats?: string[];
+    spatialRefId: string;
+  }
+) {
+  const { formats, spatialRefId } = options;
+
+  const { itemId, layerId } = parseDatasetId(datasetId);
+
+  const query = new SearchQueryBuilder()
+    .match(`exportItem:${itemId}`)
+    .in("typekeywords")
+    .and()
+    .match(`exportLayer:${layerId ? `0${layerId}` : null}`)
```

Comment:
not sure if this `0` belongs here. I ported it over from the old code, but looks like it could have been a mistake @rgwozdz .

Reply:
looks like a mistake to me

Reply:
crap, it might not be.

Reply:
Not a mistake. See here: https://github.com/Esri/hub.js/pull/472